addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -87,6 +88,38 @@
             </field>
         </record>
 
+        <record id="hr_expense_form_view" model="ir.ui.view">
+            <field name="name">hr.expense.form</field>
+            <field name="model">hr.expense</field>
+            <field name="inherit_id" ref="hr_expense.hr_expense_form_view_without_header"/>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="/form/*" position="before">
+                    <header>
+                        <!-- Buttons when there is no attachment with the expense -->
+                        <button name="submit_expenses" string="Submit to Manager" type="object" attrs="{'invisible': ['|', ('attachment_number', '=', 0), ('state', 'not in', 'draft')]}" class="oe_highlight o_expense_submit"/>
+                        <button name="upload_document" string="Upload a document" type="object" attrs="{'invisible': ['|', ('attachment_number', '=', 0), ('state', 'not in', 'draft')]}" class="o_upload_document"/>
```

Comment:
The upload document feature has been done here : https://www.odoo.com/web#id=36712&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
So it has to be removed from this task. Sorry there was some miscommunication about it in this task.